src/gui/MainWindow.cpp

```diff
@@ -1611,5 +1611,36 @@ void MainWindow::onProjectFileNameChanged()
 	this->resetWindowTitle();
 }
 
+CustomQMdiArea::CustomQMdiArea(QWidget* parent) :
+	QMdiArea(parent),
+	m_isBeingMoved(false)
```

Comment:
We should initialize all the member variables.
```suggestion
	m_isBeingMoved(false),
	m_lastX(0),
	m_lastY(0)
```